hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/MappingModelCreationProcess.java

```diff
@@ -15,37 +15,44 @@
 import org.hibernate.metamodel.mapping.NonTransientException;
 import org.hibernate.metamodel.spi.RuntimeModelCreationContext;
 import org.hibernate.persister.entity.EntityPersister;
+import org.hibernate.query.sqm.function.SqmFunctionRegistry;
 
 /**
  * @author Steve Ebersole
  */
 public class MappingModelCreationProcess {
+	private final String EOL = System.getProperty( "line.separator" );
```

Comment:
from Javadoc, `System.lineSeparator` always returns the same value as above.

Reply:
Pushed upstream with suggested changes